src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -110,13 +141,31 @@ export default defineComponent({
             return { 'border-left': `2px solid ${selectedColor.value}` }
         })
 
-        const rows = computed<DlTableRow[]>(() => {
-            return items.value?.map((i: DlLabelPickerItem) => ({
-                ...i,
-                name: i.displayLabel
-            }))
-        })
+        const mapObjects = (
+            item: DlLabelPickerItem,
+            callback: (obj: DlLabelPickerItem) => DlLabelPickerItem
+        ) => {
+            const mappedItem: DlLabelPickerItem = callback(item)
+
+            if (item.children && item.children.length > 0) {
+                mappedItem.children = item.children.map((child) =>
+                    mapObjects(child, callback)
+                )
+            }
+            return mappedItem
+        }
 
+        const mapItems = ref<DlTableRow[]>(
```

Comment:
Why do we need to hold this in a ref ? 
cant this be a computed getter instead of rows ?